examples/AN00156_i2c_master_example/doc/rst/AN00156.rst

```diff
@@ -1,4 +1,4 @@
-.. include:: ../../README.rst
+.. |I2C| replace:: I\ :sup:`2`\ C
```

Comment:
Why this change?

Reply:
That include is not needed for the string replacement as far as I can see, but I will revert that change to be sure

Reply:
> That include is not needed for the string replacement as far as I can see, but I will revert that change to be sure

My question wasn't specific to the string replacement, and I don't know as much about the ReadMe file contents as you probably do by now.  By removing the include, it looks like the ReadMe file will contain only the I2S string and will be empty otherwise.  Have I misunderstood.

Reply:
No, I had a deeper look, it looks like the README is included and it is part of that doc examples/AN00156_i2c_master_example/doc/rst/AN00156.rst

I reverted all the changes. I am not sure if we should remove xdoc custom directives from that doc as well, but there are many of them. I would do it in a separate PR